.github/workflows/scripts/oidc-helper.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```

Comment:
I dont think this script is necessary. For checking URL you have the wait-for-url script